pareto/utilities/visualize.py

```diff
@@ -0,0 +1,130 @@
+#####################################################################################################
+# PARETO was produced under the DOE Produced Water Application for Beneficial Reuse Environmental
+# Impact and Treatment Optimization (PARETO), and is copyright (c) 2021-2024 by the software owners:
+# The Regents of the University of California, through Lawrence Berkeley National Laboratory, et al.
+# All rights reserved.
+#
+# NOTICE. This Software was developed under funding from the U.S. Department of Energy and the U.S.
+# Government consequently retains certain rights. As such, the U.S. Government has been granted for
+# itself and others acting on its behalf a paid-up, nonexclusive, irrevocable, worldwide license in
+# the Software to reproduce, distribute copies to the public, prepare derivative works, and perform
+# publicly and display publicly, and to permit others to do so.
+#####################################################################################################
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+
+def plot_network(m):
+    G = nx.Graph()
+    for i in m.s_L:
+        G.add_node(i)
+
+    piping_acrs = []
+    for i in m.s_LLA:
+        piping_acrs.append(i)
+    trucking_arcs = []
+    for i in m.s_LLT:
+        trucking_arcs.append(i)
```

Comment:
```suggestion
```
It seems that these lines of code can be deleted since trucking arcs are not plotted. However, you might be able to add an option to the function to allow the user to include trucking arcs in addition to or instead of piping arcs if desired. Does networkx allow you to draw two edges between nodes in a graph? Suppose two PW network elements are connected by a trucking arc and a piping arc. Could we then create a figure where there are two edges between these nodes, one for trucking and one for piping? We would need the edges to look different in some manner - e.g., we could use dashed lines for trucking arcs and solid lines for piping arcs.

Reply:
The option to plot trucking arcs is now implemented; resolving.